code/modules/unit_tests/projectiles.dm

```diff
@@ -3,3 +3,27 @@
 		var/obj/projectile/projectile = path
 		if(initial(projectile.movement_type) & PHASING)
 			Fail("[path] has default movement type PHASING. Piercing projectiles should be done using the projectile piercing system, not movement_types!")
+
+/datum/unit_test/gun_go_bang/Run()
+	// test is for a ballistic gun that starts loaded + chambered
+	var/obj/item/gun/test_gun = allocate(/obj/item/gun/ballistic/automatic/pistol)
+	var/mob/living/carbon/human/victim = allocate(/mob/living/carbon/human)
+	var/mob/living/carbon/human/gunner = allocate(/mob/living/carbon/human)
+
+	ADD_TRAIT(victim, TRAIT_RESISTLOWPRESSURE, INNATE_TRAIT) // so pressure damage doesn't throw off our damage check
```

Comment:
Man, I really need to make the testing place not in space by default.

Reply:
YEAH, wasted 30 minutes figuring out where that extra 2.4 brute was coming from

Reply:
:sob: